From a5516a9661bce6cbbc2a801e615dcec1fa4e89e2 Mon Sep 17 00:00:00 2001
From: David Heidelberg <david@ixit.cz>
Date: Tue, 18 Nov 2025 14:56:11 +0100
Subject: [PATCH 108/211] EDITME: cover title for rpmsg-qcom-glink

Abandoned, discussion here:
  https://lore.kernel.org/all/20230117142414.983946-1-caleb.connolly@linaro.org/

Downstream note:

    FIX: rpmsg: glink: fix wakeup nullptr

    Somehow rpdev can be null???

    weird...

# Describe the purpose of this series. The information you put here
# will be used by the project maintainer to make a decision whether
# your patches should be reviewed, and in what priority order. Please be
# very detailed and link to any relevant discussions or sites that the
# maintainer can review to better understand your proposed changes. If you
# only have a single patch in your series, the contents of the cover
# letter will be appended to the "under-the-cut" portion of the patch.

# Lines starting with # will be removed from the cover letter. You can
# use them to add notes or reminders to yourself. If you want to use
# markdown headers in your cover letter, start the line with ">#".

# You can add trailers to the cover letter. Any email addresses found in
# these trailers will be added to the addresses specified/generated
# during the b4 send stage. You can also run "b4 prep --auto-to-cc" to
# auto-populate the To: and Cc: trailers based on the code being
# modified.

To: Bjorn Andersson <andersson@kernel.org>
To: Mathieu Poirier <mathieu.poirier@linaro.org>
Cc: linux-arm-msm@vger.kernel.org
Cc: linux-remoteproc@vger.kernel.org
Cc: linux-kernel@vger.kernel.org
Cc: phone-devel@vger.kernel.org
Signed-off-by: David Heidelberg <david@ixit.cz>

--- b4-submit-tracking ---
# This section is used internally by b4 prep for tracking purposes.
{
  "series": {
    "revision": 2,
    "change-id": "20251118-rpmsg-qcom-glink-8aab10623ff5",
    "prefixes": [],
    "from-thread": "20230117142414.983946-1-caleb.connolly@linaro.org"
  }
}
-- 
2.55.0

